Deploy/kubernetes/pv/log.pv.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: log-pv
+  labels:
+    type: log
+spec:
+  capacity:
+    storage: 5Gi
```

Comment:
Actually, if you are just using hostpath for ur persistent data, there is really no need to use PV and PVC.  You just declare Pod using hostpath, e.g.

```yaml
apiVersion: v1
kind: Pod
metadata:
  name: hostpath
spec:
  containers:
    - name: hostpath
      image: index.caicloud.io/caicloud/hostpath
      imagePullPolicy: Always
      volumeMounts:
        - mountPath: "/etc"
          name: etc
  volumes:
    - name: etc-caicloud
      hostPath:
        path: "/etc"
```